web/server/vue-cli/e2e/specs/reports.js

```diff
@@ -1,3 +1,10 @@
+function getSelectedItemText(browser, element, cb) {
+  const elementId = element.ELEMENT ||
+    element["element-6066-11e4-a52e-4f735466cecf"];
```

Comment:
The web element identifier is a constant with the string `element-6066-11e4-a52e-4f735466cecf` (https://www.w3.org/TR/2015/WD-webdriver-20150827/). So in case of Chrome the ELEMENT field will be set but in case of Firefox this constant will be set.